packages/cloudflare/src/index.ts

```diff
@@ -123,10 +131,15 @@ export default function createIntegration(args?: Options): AstroIntegration {
 							cloudflareModulePlugin,
 						],
 					},
+					integrations: [astroWhen()],
```

Comment:
It should check if the integration is already in the project to avoid duplicates. It would still work but re-do work unnecessarily

Reply:
Not blocking, once I understand the issue and the possible solution, I'll can sent a follow-up PR.
However this text from the API reference, let me think, that `updateConfig` does not add integrations twice 🤔

> Any config you provide will be merged with the user config + other integration config updates.